conv.go

```diff
@@ -3,40 +3,169 @@
 package golgi
 
 import (
+	"fmt"
+
 	"github.com/chewxy/hm"
-	G "gorgonia.org/gorgonia"
+	"gorgonia.org/gorgonia"
 	"gorgonia.org/tensor"
 )
 
+// ConsConv is a Conv construction function. It takes a gorgonia.Input that has a *gorgonia.Node.
+func ConsConv(in gorgonia.Input, opts ...ConsOpt) (retVal Layer, err error) {
+	x := in.Node()
+	if x == nil {
+		return nil, fmt.Errorf("ConsConv expects a *Node. Got input %v of  %T instead", in, in)
+	}
+
+	inshape := x.Shape()
+	if inshape.Dims() != 4 || inshape.Dims() == 0 {
+		return nil, fmt.Errorf("Expected shape is either a vector or a matrix, got %v", inshape)
+	}
+
+	l := &Conv{
+		act:         gorgonia.Rectify,
+		kernelShape: tensor.Shape{5, 5},
```

Comment:
default kernel shape should be documented

Reply:
I'll do that